include/aroma/hid/hid.h

```diff
@@ -153,5 +153,6 @@ byte libaroma_hid_get(LIBAROMA_HID_EVENTP e);
  */
 byte libaroma_hid_config(char * name,char * svalue,dword dvalue);
 
+void libaroma_hid_restart();
```

Comment:
you should enable `-Wstrict-prototypes`. While `void` is optional for C++, in C it has a different, quite dangerous semantic.